08-transport.Rmd

```diff
@@ -229,24 +247,26 @@ qtm(zones, c("all", "all_dest"), fill.palette = viridis::plasma(5),
 
 ## Desire lines
 
-Unlike zones, which represent trip origins and destinations, desire lines represent where people *desire* to go between zones.
+Unlike zones, which represent trip origins and destinations, desire lines connect the centroid of the origin and the destination zone, and thereby represent where people *desire* to go between zones.
 If it were not for obstacles such as buildings and windy roads, people would travel in a 'bee-line' from one place to the next, explaining why desire lines are straight (we will see how to convert desire lines into routes in the next section).
 
 We have already loaded data representing desire lines in the dataset `od`.
 This origin-destination (OD) data frame object represents the number of people traveling between the zone represented in `o` and `d`, as illustrated in Table \@ref(tab:od).
-This subset of the data was using attribute operations described in Chapter \@ref(attr), to arrange the OD data by all trips and then filter-out only the top 5:
+To arrange the OD data by all trips and then filter-out only the top 5, type (please refer to Chapter \@ref(attr) for a detailed description of non-spatial attribute operations):
 
 ```{r}
 od_top5 = arrange(od, desc(all)) %>% 
   top_n(5, wt = all)
 ```
 
+<!-- I guess that geocode_1 refers to origin and geocode_2 refers to destination, but in the text you talk of columns o and d, pls change acoordingly. -->
 ```{r od, echo=FALSE}
 od_top5 %>% 
   knitr::kable(caption = "Sample of the origin-destination data stored in the data frame object `od`. These represent the top 5 most common desire lines between zones in the study area.")
 ```
 
-This table provides a snapshot of Bristolian travel patterns.
+<!-- How can you infer intrazonal trips from the table? Do you mean interzonal, i.e., trips between zones? -->
```

Comment:
How can you infer intrazonal trips from the table? Do you mean interzonal, i.e., trips between zones? Ah, now I see, row 5 refers to an intrazonal trip! Ok, then refer to row 5 in the text, otherwise the reader is forced to search and to find out that there are indeed intrazonal pairs.

Reply:
OK good point - pls clarify it (the table changed, that may explain some confusion here).

Reply:
ok, done that.